src/source.h

```diff
@@ -17,7 +18,7 @@ namespace librealsense
     class LRS_EXTENSION_API frame_source
     {
     public:
-        using archive_id = std::pair< rs2_stream, rs2_extension >;
+        using archive_id = std::tuple< rs2_stream, int, rs2_extension >;
```

Comment:
Can we add a name for the variable or a comment for what int is used for?
`using archive_id = std::tuple< rs2_stream, int index, rs2_extension >;`

Reply:
Can't add a name, I will add a comment